### PR TITLE
search portal name change

### PR DESCRIPTION
**Pull request by alias01**

### CODE REVIEW

File: src/components/templates/TemplateSearchPortal.vue [hunk 1]
```diff
@@ -1,5 +1,45 @@
...
+            <hr />
+            <div class="so-priority">
+                <p>High : {{ summary.high }}</p>
+                <p>Low : {{ summary.low }}</p>
+                <p>Medium : {{ summary.medium }}</p>
+            </div>
+            <hr />
+            <div class="so-status">
+                <p>
+                    <span>Not Set :</span> <span>{{ summary.status[0] }}</span>
```
Comment (arrogant-ant) [inline src/components/templates/TemplateSearchPortal.vue:20]: We can remove this as these number will always be zero

File: src/components/templates/TemplateSearchPortal.vue [hunk 2]
```diff
@@ -1,5 +1,45 @@
...
+                <p>
+                    <span>Registered :</span>
+                    <span>{{ summary.status[1] }}</span>
+                </p>
+                <p>
+                    <span>Processing :</span>
+                    <span>{{ summary.status[2] }}</span>
+                </p>
+                <p>
+                    <span>Suggested : </span>
```
Comment (arrogant-ant) [inline src/components/templates/TemplateSearchPortal.vue:31]: We can remove this as these number will always be zero

File: src/components/templates/TemplateSearchPortal.vue [hunk 3]
```diff
@@ -1,5 +1,45 @@
...
+                    <span>{{ summary.status[3] }}</span>
+                </p>
+                <p>
+                    <span>Accepted : </span>
```
Comment (arrogant-ant) [inline src/components/templates/TemplateSearchPortal.vue:35]: We can remove this as these number will always be zero

File: src/components/templates/TemplateSearchPortal.vue (hunk 2 above)
Comment (arrogant-ant) [reply src/components/templates/TemplateSearchPortal.vue:31]: It gets filtered from backend

File: src/components/templates/TemplateSearchPortal.vue [hunk 4]
```diff
@@ -302,11 +376,57 @@ export default {
         toSrp(lat, lng) {
             this.$emit('toSrp', lat, lng);
         },
+
+        showSummary() {
+            console.log('summary');
```
Comment (arrogant-ant) [inline src/components/templates/TemplateSearchPortal.vue:381]: do we need log?

File: src/components/templates/TemplateSearchPortal.vue [hunk 5]
```diff
@@ -197,15 +237,17 @@ import AtomTextarea from '../atoms/AtomTextarea.vue';
...
+    name: 'TemplateSearchPortal',
     components: {
         AtomTextarea,
         AtomSelectInput,
         AtomDatePicker,
         AtomInput,
+        AtomButton,
     },
     props: {
         lists: {
```
Comment (arrogant-ant) [inline src/components/templates/TemplateSearchPortal.vue:253]: can we rename this?

**Review by arrogant-ant — CHANGES_REQUESTED** [review]

File: src/components/templates/TemplateSearchPortal.vue (hunk 3 above)
Comment (alias01) [reply src/components/templates/TemplateSearchPortal.vue:35]: done

File: src/components/templates/TemplateSearchPortal.vue (hunk 4 above)
Comment (alias01) [reply src/components/templates/TemplateSearchPortal.vue:381]: done

File: src/components/templates/TemplateSearchPortal.vue (hunk 2 above)
Comment (alias01) [reply src/components/templates/TemplateSearchPortal.vue:31]: done

File: src/components/templates/TemplateSearchPortal.vue (hunk 1 above)
Comment (alias01) [reply src/components/templates/TemplateSearchPortal.vue:20]: done

**Review by arrogant-ant — APPROVED** [review]

**Pull request merged**